### DISCUSSION TITLE
Netlify Deploy

### DISCUSSION THREAD

**Discussion by OlaoyeVic** [codesandbox/codesandbox-client#6131] [Q&A]
I tried deploying my app with netlify, but after the deployment it is not displaying anything; more like a blank page. Please can you help with any assistance?
Here's a link to my codesandbox;
[https://codesandbox.io/s/infallible-elion-2jqx0](url)

**Comment by willklein** [comment]
![image](https://user-images.githubusercontent.com/1075861/174457358-4ed00aa0-1708-4dbe-81ae-8331d1841941.png)

The sandbox link doesn't work, but FYI, you can View Logs when deploying and look for errors.
